space_dodge/drawing/pause_menu/settings.py

```
@@ -1,26 +1,24 @@
 
 import pygame
 import pygame_widgets
-import time
-
-from pygame_widgets.slider import Slider
-from pygame_widgets.textbox import TextBox
-
 from classes.button import Button
 from file_handling.constants_and_file_loading import (
     WINDOW, pause_background, muteImage, unmuteImage, PAUSE_FONT, pause_time, x_button_icon)
+from file_handling.constants_and_file_loading import logger
```

Flake8 (7.2.0)

[error] 'file_handling.constants_and_file_loading.logger' imported but unused

(F401)